css/json.human.css

```diff
@@ -1,27 +1,48 @@
+@import url(http://fonts.googleapis.com/css?family=Inconsolata);
+
 .jh-root, .jh-type-object, .jh-type-array, .jh-key, .jh-value, .jh-root tr{
  -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
  -moz-box-sizing: border-box;    /* Firefox, other Gecko */
  box-sizing: border-box;         /* Opera/IE 8+ */
+ font-family: 'Inconsolata' ;
```

Comment:
I've moved it to demo.css itself. Don't want to imply user's to make another server request just to get font css. They can follow our approach now, they can declare this in their own page's css and get it working.